driver/kms_driver_test.go

```diff
@@ -58,7 +58,7 @@ var _ = Describe("KmsDriver", func() {
 
 	It("replicates a given kms key to another region", func() {
 		driverConfig := resources.KmsReplicateKeyDriverConfig{
-			KmsKeyId:     multiregionKmsKeyId,
+			KmsKeyId:     multiRegionKeyReplicationTest,
```

Comment:
This test triggers the cleanup of the replicated key in line 81. That makes the private AMI tests in the copy_ami_driver_tests flakey. Therefore we have to separate the used multiregion kms keys here.